sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -2,7 +2,7 @@ namespace Azure.Iot.Hub.Service
 {
     public enum BulkIfMatchPrecondition
     {
-        Unconditional = 0,
```

Comment:
Why is this being changed? The bulk APIs don't have an unconditionalIfMatch option, do they?

Reply:
you are correct, I am reverting this change. it can be misleading.